platform/ext/target/nordic_nrf/nrf5340/target_cfg.c

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2018-2020 Arm Limited. All rights reserved.
+ * Copyright (c) 2020 Nordic Semiconductor ASA.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "target_cfg.h"
+#include "Driver_Common.h"
+#include "platform_description.h"
+#include "device_definition.h"
+#include "region_defs.h"
+#include "tfm_secure_api.h"
+#include "tfm_plat_defs.h"
+
+#include "log/tfm_assert.h"
+#include "log/tfm_log.h"
```

Comment:
Those two do not seem to be needed here.

Reply:
I guess, same as for any other inclusions - let's add them on an on-demand basis
Removed now.